src/ext_manifest.c

```diff
@@ -114,20 +113,11 @@ static int ext_man_build(const struct module *module,
 		goto out;
 	}
 
-	/* alloc buffer for ext_man */
-	buffer = calloc(1, ext_man.full_size);
-	if (!buffer) {
-		ret = -ENOMEM;
-		goto out;
-	}
-
 	/* fill buffer with ext_man and section content */
-	memcpy(buffer, &ext_man, ext_man.header_size);
+	memcpy(*dst_buff, &ext_man, ext_man.header_size);
```

Comment:
set aside memcpy_s(), this doesn't seems right. `*dst_buff` has only space for a pointer. How can we sure there is enough size for ext_man.header_size? I think, this is good reason that we need to move toward memcpy_s()

Reply:
You can't drop the `calloc`, you still need something like `*dst_buffer = calloc(...`

Reply:
Yeah, it does not look like the dest buf is being allocated.

Reply:
Yes, apologies, I did the CI changes second and forgot to test rimage. Will fix